plugins/versionpress/tests/Unit/QueryLanguageUtilsTest.php

```diff
@@ -101,8 +104,7 @@ public function ruleAndQueryProvider()
     public function queryLanguageUtilsCreatesCorrectRules($query, $expectedRules)
     {
         $rules = QueryLanguageUtils::createRulesFromQueries($query);
-        // Perform case insensitive match
-        $this->assertEquals($expectedRules, $rules, '', 0, 10, false, true);
+        $this->assertEquals($expectedRules, $rules);
```

Comment:
👍  this is a very welcome change, basically making tests actually test something :)